tests/main/interfaces-system-observe/task.yaml

```
@@ -63,6 +65,19 @@ execute: |
         su -l -c "test-snapd-system-observe-consumer.dbus-introspect" test | MATCH "$expected"
     fi
 
+    echo "Snap is able is able to query systemd properties"
+    testsnap.cmd busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.DBus.Properties \
+        GetAll s org.freedesktop.systemd1.Manager
+
+    if not os-query is-trusty; then
```

This is why spread is failing for 14.04
```suggestion
    if not os.query is-trusty; then
```